x/ccv/provider/migrations/v7/legacy_params.go

```diff
@@ -1,4 +1,4 @@
-package v6
+package v7
```

Comment:
Do we want to keep the old `legacy_params.go` file under `v6` as well in case someone wants to use it to go from 5 to 6?

Reply:
Talked offline. Resolving this because it was based on my misunderstanding.